pokecli.py

```diff
@@ -208,7 +209,13 @@ def main():
         logger.log('[x] Starting PokemonGo Bot....', 'green')
 
         while True:
-            bot.take_step()
+			try:
+				bot.take_step()
+			except NotLoggedInException:
+				logger.log('pgoapi is reporting that you are no longer logged in. Reinitializing...', 'red')
+				# re-initialize the bot wit the same config.
+				bot = PokemonGoBot(config)
```

Comment:
@kennycoc I know about the garbage collector. I was thinking that a method to reload config will be re-usable. I want to make the bot reload config automatically when it receives a USR2 signal through `kill -USR2 <bot_pid>`.